data-model.d.ts

```diff
@@ -73,6 +73,10 @@ export interface Literal {
      * @link http://tools.ietf.org/html/bcp47
      */
     language: string;
+    /**
+     * the direction of the language-tagged string.
+     */
+    direction?: 'ltr' | 'rtl' | '' | null;
```

Comment:
Ah, on second (third?) thought, can we remove the empty string from allowed values. I know you mention falsy values above, but it's still a string and not one expected by the spec and not the same as `null` or `undefined`

```suggestion
    direction?: 'ltr' | 'rtl' | null;
```